packages/fluentui/docs/src/components/ComponentDoc/ComponentControls/ComponentControls.tsx

```diff
@@ -69,7 +69,7 @@ const ComponentControls: React.FC<ComponentControlsProps> = props => {
             aria-label={toolbarAriaLabel || null}
             items={[
               {
-                icon: { name: 'code', style: { width: '20px', height: '20px' } },
+                icon: <Icon name={'code'} {...{ style: { width: '20px', height: '20px' } }} />,
```

Comment:
Any reason to actually use spread for style instead of direct using the prop/attribute?

```jsx
icon: <Icon name="code" style={{ width: '20px', height: '20px' }} />,
```

Reply:
This was auto generated by the codemods, which basically remove the name prop and spread the rest of the object on the component.